Source/WebCore/rendering/RenderTheme.cpp

```diff
@@ -701,25 +701,39 @@ OptionSet<ControlStyle::State> RenderTheme::extractControlStyleStatesForRenderer
     return states;
 }
 
-ControlStyle RenderTheme::extractControlStyleForRenderer(const RenderBox& box) const
+static const RenderObject* adjustRendererForAppearance(const RenderObject& renderObject)
```

Comment:
I would call this something like `effectiveRendererForControlStyle`. The current name makes it seem like this method changes renderer state.